client/client-multi/client-cpp/kaa/channel/IKaaChannelManager.hpp

```diff
@@ -18,13 +18,27 @@
 #define IKAACHANNELMANAGER_HPP_
 
 #include <list>
+#include <string>
 
 #include "kaa/channel/IDataChannel.hpp"
 #include "kaa/channel/ITransportConnectionInfo.hpp"
 #include "kaa/channel/connectivity/IConnectivityChecker.hpp"
 
 namespace kaa {
 
+/**
+ * The structure represents connection information in readable format.
+ */
+struct EpConnectionInfo {
```

Comment:
I'd prefer the current one. If there is a need to add some non-ip-related data (for example the type of the server), this name will be better fit.